WasatchNET/AndorSpectrometer.cs

```diff
@@ -30,10 +30,11 @@ public class AndorSpectrometer : Spectrometer
         int yPixels;
 
         //see page 330 of Andor SDK documentation
-        const int DRV_SUCCESS = 20002;
+        public const int DRV_SUCCESS = 20002;
 
         // not sure where this comes from...ask Caleb - TS
-        const int SHUTTER_SPEED_MS = 35;
+        const int SHUTTER_SPEED_MS = 50;
```

Comment:
Why did we change this?

Reply:
I believe from troubleshooting when we were trying to get the area scan working

Reply:
Okay.  @rc14193 , could you update that in Wasatch.PY when you get a chance.